CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
 
-set(CMAKE_CXX_STANDARD 17)
```

Comment:
@JiangfengLi Why have these lines been removed?

Reply:
I thought you told me to remove that

Reply:
@JiangfengLi I meant the blank lines..

Reply:
Go ahead and do 

```
git checkout master -- CMakeLists.txt
```

Reply:
(Followed by `git push`)